vars/transportRequestUploadFile.groovy

```diff
@@ -257,19 +257,18 @@ void call(Map parameters = [:]) {
 
                         echo "[INFO] Uploading application '${configuration.applicationName}' to transport request '${configuration.transportRequestId}'."
 
-                        cm.uploadFileToTransportRequestCTS(
-                            configuration.changeManagement.cts?.nodeDocker ?: [:],
-                            configuration.transportRequestId,
-                            configuration.changeManagement.endpoint,
-                            configuration.changeManagement.client,
-                            configuration.applicationName,
-                            configuration.applicationDescription,
-                            configuration.abapPackage,
-                            configuration.changeManagement.cts.osDeployUser,
-                            configuration.changeManagement.cts.deployToolDependencies,
-                            configuration.changeManagement.cts.npmInstallOpts,
-                            configuration.changeManagement.cts.deployConfigFile,
-                            configuration.changeManagement.credentialsId)
+                        transportRequestUploadCTS(script: script,
```

Comment:
How is piperExecuteBin called here?

Reply:
`transportRequestUploadCTS` is a Jenkins groovy pipeline step which indirects to the GO implementation via the 

`piperExecuteBin(parameters, STEP_NAME, METADATA_FILE, credentials)`

call

Reply:
Ah, so this is a call to another step, got it.